client/account-recovery/reset-password/reset-password-form/test/index.jsx

```diff
@@ -37,32 +38,25 @@ describe( 'ResetPasswordForm', () => {
 		const wrapper = shallow(
 			<ResetPasswordFormComponent
 				resetOptions={ exampleResetOptions }
+				translate={ identity }
 			/>
 		);
 
-		expect( wrapper ).to.have.state( 'isSubmitting' ).to.be.false;
 		expect( wrapper.find( ResetOptionSet ) ).to.have.length( 2 );
-
 		expect( wrapper.find( '.reset-password-form__submit-button' ).prop( 'disabled' ) ).to.be.ok;
 	} );
 
 	context( 'fields', () => {
 		useFakeDom();
 
-		it( 'should be disabled while submitting', function() {
+		it( 'should be disabled while isRequesting is on.', function() {
 			const wrapper = mount(
 				<ResetPasswordFormComponent
 					resetOptions={ exampleResetOptions }
+					isRequesting={ true }
+					translate={ identity }
```

Comment:
I can totally see how defaulting the `translate` prop can eliminate the duplication from the tests :) 
My concern was that `translate` is marked as `PropTypes.func.isRequired` from the property type declaration, and defaulting it could defeat the purpose. Another way to put it could be removing the `isRequired` and default it as `identity` as it used to be. However, I'd like to emphasize that `translate` is something we must pass in from outside, so I'd like to keep it this way here :)